src/woo_publications/publications/tests/test_admin_logging.py

```diff
@@ -175,6 +175,142 @@ def test_admin_update(self):
 
             self.assertEqual(update_log.extra_data, expected_data)
 
+    def test_admin_update_revoke_published_documents_when_revoking_publication(self):
+        assert not TimelineLogProxy.objects.exists()
+        ic, ic2 = InformationCategoryFactory.create_batch(2)
+        organisation = OrganisationFactory.create(is_actief=True)
+        with freeze_time("2024-09-27T00:14:00-00:00"):
+            publication = PublicationFactory.create(
+                publisher=organisation,
+                verantwoordelijke=organisation,
+                opsteller=organisation,
+                informatie_categorieen=[ic, ic2],
+                officiele_titel="title one",
+                verkorte_titel="one",
+                omschrijving="Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+            )
+            published_document = DocumentFactory.create(
+                publicatie=publication,
+                publicatiestatus=PublicationStatusOptions.published,
+                identifier="http://example.com/1",
+                officiele_titel="title",
+                creatiedatum="2024-10-17",
+            )
+            concept_document = DocumentFactory.create(
+                publicatie=publication,
+                publicatiestatus=PublicationStatusOptions.concept,
+            )
+            revoked_document = DocumentFactory.create(
+                publicatie=publication,
+                publicatiestatus=PublicationStatusOptions.revoked,
+            )
+
+        reverse_url = reverse(
+            "admin:publications_publication_change",
+            kwargs={"object_id": publication.id},
+        )
+
+        response = self.app.get(reverse_url, user=self.user)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(TimelineLogProxy.objects.count(), 1)
+
+        with self.subTest("read audit logging"):
+            log = TimelineLogProxy.objects.get()
+
+            expected_data = {
+                "event": Events.read,
+                "acting_user": {
+                    "identifier": self.user.id,
+                    "display_name": self.user.get_full_name(),
+                },
+                "_cached_object_repr": "title one",
+            }
+
+            self.assertEqual(log.extra_data, expected_data)
+
+        form = response.forms["publication_form"]
+        form["publicatiestatus"].select(text=PublicationStatusOptions.revoked.label)
+
+        with freeze_time("2024-09-28T00:14:00-00:00"):
+            response = form.submit(name="_save")
+
+        self.assertEqual(response.status_code, 302)
```

Comment:
I get a message saying there are validation errors, but I don't see what's wrong/not valid. This test however seems to correctly indicate a succesful revocation :thinking: